x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/store/builders.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// import { MANAGEMENT_DEFAULT_PAGE, MANAGEMENT_DEFAULT_PAGE_SIZE } from '../../../common/constants';
+import { MANAGEMENT_DEFAULT_PAGE, MANAGEMENT_DEFAULT_PAGE_SIZE } from '../../../common/constants';
+import { createUninitialisedResourceState } from '../../../state';
+import { HostIsolationExceptionsPageState } from '../types';
+
+// @ts-ignore
```

Comment:
Perfect example of why `ts-expect-error` works out here. If you replace `ts-ignore` with `ts-expect-error` you would get an error where TypeScript tells you there is no error here and you should remove this line.

<img width="850" alt="Screen Shot 2021-09-29 at 10 11 42 AM" src="https://user-images.githubusercontent.com/1151048/135307764-48edd9df-9ff3-4d1f-aa06-e315bde258eb.png">

With `ts-ignore` we have to check each and every instance in a pull request to ensure it's actually ignoring a valid error, where with `ts-expect-error` we know during code reviews there _must_ be an error otherwise you wouldn't be able to checkin.

Overall though, remove this line as it's not doing anything.

Reply:
Good point about the ts-expect-error. The comment was a wrong-paste of some sort. There's no reason to have it there as you say.